### ISSUE TITLE
Hyperparameters for reproducing point cloud classification

### ISSUE THREAD

**Issue by andreimano** [StefanosPert/Equivariant_Optimization_CR#3]
Hi there!

Once again, thanks a lot for making your code publicly available!

I am unable to reproduce the experiments from Figure 2 - most of the runs converge at ~84% accuracy for DG-CNN when using the default hyperparameters from the readme file.

Could you please provide the hyperparameters for reproducing the experiments (incl. ablations)?

Thank you!

**Comment by StefanosPert** [comment]
Is the test accuracy or the test avg. accuracy converging to ~84%? We were reporting the test accuracy in the above figure

**Comment by andreimano** [comment]
Neither of them typically go beyond 0.9 - for instance, in this run, the instance (test) accuracy goes to 87%, and the avg accuracy is ~84%

<img width="1867" alt="image" src="https://github.com/user-attachments/assets/c12d9964-b4cf-4801-8f25-91721bcf9f66">

From the paper, it looks like the model should surpass 90% accuracy:

<img width="720" alt="image" src="https://github.com/user-attachments/assets/9bd91e96-2e01-4b18-8553-31a2dadb8a97">

But I have been unable to reproduce this in any of my runs - both the purely equivariant model and your approach stay in the 84%-88% accuracy range (depending on the seed). Therefore, I may be doing something wrong and missed some hyperparameters.

**Comment by StefanosPert** [comment]
The test accuracy should converge around 88.5% (I am currently retraining to verify that). In the original Figure 2 from the arxiv there was a mistake, we corrected it in our Neurips camera-ready version in [openreview](https://openreview.net/forum?id=tWkL7k1u5v&referrer=%5BAuthor%20Console%5D(%2Fgroup%3Fid%3DNeurIPS.cc%2F2024%2FConference%2FAuthors%23your-submissions)). So the corrected figure looks like this:
![updated_vn-dgcnn_figure](https://github.com/user-attachments/assets/8be6eb2f-99e8-48da-87e5-63359f16c659)
We will also add it to the arxiv once we update it to match our camera-ready version. In general, the improvements gained using our method on vn-dgcnn are significantly smaller than those gained with the pointnet architecture. We are currently investigating the factors contributing to this difference and we are open to discussing more about it if you have suggestions for this research direction.

**Comment by andreimano** [comment]
That seems more in line with my experiments; thanks a lot! Please let me know what the final result of your current run is 😄 .

Regarding the investigation - yes, I find your paper very interesting, and I've been looking into similar things lately :). I will be at NeurIPS this year. We could have a face-to-face chat there. Otherwise, we could discuss via e-mail.

Thanks a lot for your responsiveness! I greatly appreciate it :)

**Comment by StefanosPert** [comment]
Thank you for your interest in our work! We can definitely discuss more about it during NeurIPS.
The training I was running converged to 88.37% test accuracy, but as you also mentioned it can vary given the random seed.
